pkgs/by-name/go/goldboot/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  fetchFromGitHub,
+  rustPlatform,
+  pkgs,
+  lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "goldboot";
+  version = "0.0.10";
+
+  meta = with lib; {
+    mainProgram = "goldboot";
+    description = "Immutable infrastructure for the desktop!";
+    homepage = "https://github.com/fossable/goldboot";
+    license = licenses.agpl3Plus;
+    sourceProvenance = with sourceTypes; [ fromSource ];
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ cilki ];
+  };
+  nativeBuildInputs = with pkgs; [ pkg-config ];
+  buildInputs = with pkgs; [
+    zstd
+    OVMF
+    qemu
+    qemu-utils
+    openssl
+  ];
```

Comment:
I thnk this part should be after `src`. It should roughtly follow the phases (think of it like this: you want to know what are your sources before how to build them)